src/main/java/sirius/biz/tenants/SAMLController.java

```diff
@@ -181,7 +181,7 @@ protected List<T> querySAMLTenants() {
 
     private void verifyUser(SAMLResponse response, UserInfo user) {
         U account = user.getUserObject(getUserClass());
-        T tenant = account.getTenant().fetchValue();
+        T tenant = account.getTenant().fetchCachedValue();
```

Comment:
**Question:** Wouldn't it be better to use the cache of the `Tenants` part here?

Reply:
It is. Extrapolates the purpose of the change/ticket though which requires additional/special testing.